sdk/compute/Azure.ResourceManager.Compute/src/autorest.md

```diff
@@ -191,6 +191,9 @@ rename-mapping:
   UpgradePolicy: VirtualMachineScaleSetUpgradePolicy
   ResourceSkuCapabilities: ComputeResourceSkuCapabilities
   ProtocolTypes: WinRMListenerProtocolType
+  VMGuestPatchRebootBehavior: VirtualMachineGuestPatchRebootBehavior
+  VMGuestPatchRebootSetting: VirtualMachineGuestPatchRebootSetting
+  VMGuestPatchRebootStatus: VirtualMachineGuestPatchRebootStatus
```

Comment:
I did not change these three because I think VmGuest is kind of a fixed phrase, it looks really weird we have both VmGuest (see above) and VirtualMachineGuest.

Reply:
According to the description from the swagger spec, we can remove `Guest` here